server/src/main/java/io/pravega/schemaregistry/storage/client/TableStore.java

```diff
@@ -183,12 +183,14 @@ protected void doStop() {
     }
 
     public CompletableFuture<List<Version>> updateEntries(String tableName, Map<byte[], VersionedRecord<byte[]>> batch) {
+        log.info("Update entries called");
         Preconditions.checkNotNull(batch);
         List<TableSegmentEntry> entries = batch.entrySet().stream().map(x -> {
             return x.getValue().getVersion() == null ?
                     TableSegmentEntry.notExists(x.getKey(), x.getValue().getRecord()) :
                     TableSegmentEntry.versioned(x.getKey(), x.getValue().getRecord(), x.getValue().getVersion().toLong());
         }).collect(Collectors.toList());
+        log.debug("Wirecommand to update entry is being called");
```

Comment:
Again, in most normal situations I don;t see why we should need these logs. If these logs have been added only for debugging the current issue, please remove them from the PR.